script.js

```diff
@@ -1,4 +1,182 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+// Please fill out the survey before submitting the pull request. Thanks!
+
+// 🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀
+// How many hours did you spend on this assignment?
+// 7
+// Please fill in one error and/or error message you received while working on this assignment.
+// the nonstop syntax issues telling me I have 1 too much }
+// What part of the assignment did you spend the most time on?
+// debugging my validity logic and also the duplicate checking
+// Comfort Level (1-5):
+//3
+// Completeness Level (1-5):
+//3
+// What did you think of this deliverable?
+// I tried. However I feel like i am limited to what I can do because I don't know too much.
+// Is there anything in this code that you feel pleased about?
+// I have the ability to set the dice amounts in this game
+
+// There are 2 players and players take turns.
+// When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+// The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+// After both players have rolled and chosen dice order, the player with the higher combined number wins.
+//
+// plan:
+// gameState 0 should be default and show nothing, but click submit to start rolling to p1
+// gameState 1, 3should show you Player's rolls and asks you to key in option 1 or option 2
+// gameState 2, 4 should show u the result of the corresponding player. it then proceeds to start the next roll
+// gameState 5 should be a winner screen, comparing the results, then submitting again should send back to state 0
+let totalDice = 0;
+let gameState = 0;
+let recentRolls = [];
+let player1Hand = "";
+let player2Hand = "";
+let playerTurn = 1;
+
+//main game
+function main(input) {
+  if (totalDice === 0) {
+    if (
+      !isNaN(input) &&
+      input !== undefined &&
+      input !== "" &&
+      input < 6 &&
+      input > 0
+    ) {
+      totalDice = input;
+      return `Total amount of Dice set to ${input}!`;
+    } else {
+      return `Please set the amount of dice first! Type in a number! Max dice is 5!`;
+    }
+  }
+  if (gameState === 0 || gameState === 2) {
+    if (!(input === "")) {
+      return `Hello ${callPlayer(
+        playerTurn
+      )}!<br>Do NOT type anything and submit to start the round!`;
+    }
+    rollDice();
+    gameState += 1;
```

Comment:
rather than using numbers for gameState, i would recommend giving them descriptive names